### PR TITLE
removed html_enc from ABC,ANAGRAM,AVERAGE,BASE64,BITSUM,CHARS

### PR DESCRIPTION
**Pull request by JSSaini07**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

We no longer need to html_enc the text returned from Goodies thanks to recent internal changes.

###### People to notify (@mention interested parties)

@moollaza

### CODE REVIEW

**Comment by daxtheduck** [comment]
@JSSaini07 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/Chars.pm [hunk 1]
```diff
@@ -51,7 +51,7 @@ handle remainder => sub {
       structured_answer => {
         data => {
             title    => $len,
-            subtitle => 'Character count: ' . html_enc($str)
+            subtitle => 'Character count: ' . $str
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Chars.pm]: Lets use string interpolation where possible e.g., `"Character count: $str"` (there's several places this should be updated).

**Comment by GuiltyDolphin** [comment]
@JSSaini07 This should be good to go once you've updated the string interpolation.

**Comment by JSSaini07** [comment]
I'll do it today :)

File: lib/DDG/Goodie/Anagram.pm [hunk 2]
```diff
@@ -95,8 +95,8 @@ handle remainder => sub {
     return $response,
       structured_answer => {
           data => {
-              title    => html_enc($response),
-              subtitle => $operation . ' ' . html_enc($word)
+              title    => $response,
+              subtitle => $operation . ' ' . $word
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Anagram.pm]: There's more interpolation that should be corrected - have a look through the diff to find them.

**Comment by GuiltyDolphin** [comment]
@JSSaini07 Thanks! :dolphin:

**Comment by JSSaini07** [comment]
Happy to help :)

**Pull request merged**